modules/microshift-ingress-controller-replace-default-certificate.adoc

```diff
@@ -0,0 +1,137 @@
+
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-ingress-controller.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-ingress-controller-replace-default-certificate_{context}"]
+= Replacing the default ingress certificate
+
+You can replace the default ingress certificate for all applications under the `.apps` subdomain. After you replace the certificate, all applications have encryption provided by the specified certificate.
+
+.Prerequisites
+
+* You must have a wildcard certificate for the fully qualified `.apps` subdomain and its corresponding private key. Each should be in a separate PEM format file.
+* The private key must be unencrypted. If your key is encrypted, decrypt it before importing it into {product-title}.
+* The certificate must include the `subjectAltName` extension showing `*.apps.<clustername>.<domain>`.
+
+.Procedure
+ 
+. In the certificate file, list the wildcard certificate as the first certificate, followed by other intermediate certificates, and then end the list with the root CA certificate. The certificate file can contain one or more certificates in a chain.
+
+. Copy the root CA certificate into an additional PEM format file.
+
+. Verify that all certificates which include `-----END CERTIFICATE-----` also end with one carriage return after that line.
+
+. Create a config map that includes only the root CA certificate that is used to sign the wildcard certificate:
++
+[source,terminal]
+----
+$ oc create configmap custom-ca \
+     --from-file=ca-bundle.crt=</path/to/example-ca.crt> \//<1>
+     -n openshift-config
+----
+<1> `</path/to/example-ca.crt>` is the path to the root CA certificate file on your local file system. For example, `/etc/pki/ca-trust/source/anchors`.
+
+. Update the cluster-wide proxy configuration with the newly created config map:
++
+[source,terminal]
+----
+$ oc patch proxy/cluster \
+     --type=merge \
+     --patch='{"spec":{"trustedCA":{"name":"custom-ca"}}}'
+----
++
+
+. Create a secret that contains the wildcard certificate chain and key:
++
+[source,terminal]
+----
+$ oc create secret tls <secret> \//<1>
+     --cert=</path/to/cert.crt> \//<2>
+// Module included in the following assemblies:
```

Comment:
The formatting is wrong from here:
<img width="853" alt="image" src="https://github.com/user-attachments/assets/53314d00-f444-45e9-863d-7c98bc60d975" />